allensdk/test/brain_observatory/behavior/test_running_processing.py

```diff
@@ -71,21 +71,36 @@ def test_get_running_df(running_data, timestamps, lowpass):
         assert np.count_nonzero(np.isnan(actual["speed"])) == 0
 
 
-def test_get_running_df_one_fewer_timestamp_check_warning():
-    with pytest.warns():
+@pytest.mark.parametrize(
+    "lowpass", [True, False]
+)
+def test_get_running_df_one_fewer_timestamp_check_warning(running_data,
+                                                          timestamps,
+                                                          lowpass):
+    with pytest.warns(UserWarning):
+        # Call with one fewer timestamp, check for a warning
         _ = get_running_df(
-            np.ndarray([1, 1, 1, 1, 1]),
-            np.ndarray([1, 2, 3, 4])
+            data=running_data,
+            time=timestamps[:-1],
+            lowpass=lowpass
         )
 
 
-def test_get_running_df_one_fewer_timestamp_check_truncation():
+@pytest.mark.parametrize(
+    "lowpass", [True, False]
+)
+def test_get_running_df_one_fewer_timestamp_check_truncation(running_data,
+                                                             timestamps,
+                                                             lowpass):
+    # Call with one fewer timestamp
     output = get_running_df(
-        np.ndarray([1, 1, 1, 1, 1]),
-        np.ndarray([1, 2, 3, 4])
+        data=running_data,
+        time=timestamps[:-1],
+        lowpass=lowpass
     )
 
-    assert len(output) == 4
+    # Check that the output is actually trimmed
+    assert len(output) == len(timestamps) - 1
```

Comment:
I would add the following asserts:

```
assert output["v_sig"] == running_data["items"]["behavior"]["encoders"]["vsig"]
assert output["v_in"] == running_data["items"]["behavior"]["encoders"]["vin"]
```

Which checks that encoder values are truncated **AND** have the expected values.